packages/schema-builder/src/index.ts

```diff
@@ -0,0 +1,3 @@
+import { SchemaBuilder } from './builder.js';
+
+export { SchemaBuilder };
```

Comment:
This can be a one liner
```suggestion
export { SchemaBuilder } from './builder.js';
```